src/signals/incident/components/ReplyForm/Notice/Notice.tsx

```diff
@@ -24,7 +19,9 @@ const Notice = ({ content, title }: NoticeProps) => (
     <Column span={12}>
       <div>
         <StyledHeading>{title}</StyledHeading>
-        <Paragraph>{content}</Paragraph>
+        <ReactMarkdown skipHtml allowedElements={['a', 'p']}>
```

Comment:
This renders unstyled markup. We have a styled markdown renderer in this project, no?

Reply:
There is a Markdown component, but the weird thing is it has the wrong styling. Both Markdown and ReactMarkdown are used in the project
<img width="1228" alt="Screenshot 2025-03-10 at 13 34 35" src="https://github.com/user-attachments/assets/5f805460-20d2-4714-b371-b99b3b8a1405" />

Reply:
I'm not sure what you mean with 'the wrong styling', but we never want to render unstyled components I think. If the `Markdown` component does not render the right styles, we can change that or add to them I guess?